dashboard/db/migrate/20170127183604_create_sign_ins.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSignIns < ActiveRecord::Migration[5.0]
+  def change
+    create_table :sign_ins do |t|
```

Comment:
Should we add `sign_in_ip` to this table, considering the `current_sign_in_ip` and `last_sign_in_ip` columns already exist in the `users` table? If not, is there a specific reason why not (e.g., we don't want to store all previous signed in IPs, only the last two)? Just want to make sure this was explicitly considered one way or another.